### PR TITLE
Enable browser builds.

### PR DESCRIPTION
**Pull request by mattmccray**
Uses browserify and babelify to assemble a bower-friendly standalone
version of Alt.

Updated bower's `main` to use the browser version, removed bower
dependencies block as they are now longer needed.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -6,13 +6,7 @@
...
   ],
   "description": "Alt is a flux implementation that is small (~4.3kb & 400 LOC), well tested, terse, insanely flexible, and forward thinking.",
-  "main": "dist/alt.js",
-  "dependencies": {
-    "es-symbol": "^1.0.1",
-    "eventemitter3": "^0.1.6",
-    "flux": "^2.0.1",
-    "object-assign": "^2.0.0"
-  },
+  "main": "dist/alt-browser.js",
```
Comment (goatslacker) [inline bower.json]: :+1:

File: src/alt-with-addons.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import Alt from './alt'
+import ReactStateMagicMixin from '../mixins/ReactStateMagicMixin'
+import ListenerMixin from '../mixins/ListenerMixin'
+import FluxyMixin from '../mixins/FluxyMixin'
+import Subscribe from '../mixins/Subscribe'
+import ActionListeners from '../utils/ActionListeners'
+import DispatcherRecorder from '../utils/DispatcherRecorder'
+import makeFinalStore from '../utils/makeFinalStore'
+
+Alt.addons = {
```
Comment (goatslacker) [inline src/alt-with-addons.js]: can you abc order these?

File: src/alt-with-addons.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+import Alt from './alt'
+import ReactStateMagicMixin from '../mixins/ReactStateMagicMixin'
```
Comment (goatslacker) [inline src/alt-with-addons.js]: abc order as well

**Comment by goatslacker** [comment]
Oh, last thing, can you base it off latest build?

**Comment by mattmccray** [comment]
What do you mean? My fork says it's up to date.

**Comment by goatslacker** [comment]
Ah, never mind then. I may not have pushed upstream. I can rebuild.

**Pull request merged**